pkg/services/rpcsrv/subscription_test.go

```diff
@@ -625,3 +625,60 @@ func TestSubscriptionOverflow(t *testing.T) {
 	finishedFlag.CompareAndSwap(false, true)
 	c.Close()
 }
+
+func TestFilteredIsValidSubscriptions(t *testing.T) {
+	priv0 := testchain.PrivateKeyByID(0)
+	var goodSender = priv0.GetScriptHash()
+
+	var cases = map[string]struct {
+		params string
+	}{
+		"tx wrong sender": {
+			params: `["transaction_added", {"unknown_sender":"` + goodSender.StringLE() + `"}]`,
+		},
+		"notification with long name": {
+			params: `["notification_from_execution", {"name":"notification_from_execution_with_long_name"}]`,
+		},
+		"execution not valid vm state": {
+			params: `["transaction_executed", {"state":"NOTHALT"}]`,
+		},
+	}
+	var s string
+	for name, this := range cases {
+		t.Run(name, func(t *testing.T) {
+			chain, rpcSrv, c, respMsgs, finishedFlag := initCleanServerAndWSClient(t)
```

Comment:
Move chain creation above a subsequent test run, because you don't need to create new chain for every test, it's enough to have a single instance. All our tests take too long to pass, so if possible, we should make new tests as fast as possible.